src/ucp/tag/eager_multi.c

```diff
@@ -29,6 +29,25 @@ ucp_proto_eager_set_middle_hdr(ucp_request_t *req, ucp_eager_middle_hdr_t *hdr)
     hdr->offset = req->send.state.dt_iter.offset;
 }
 
+static UCS_F_ALWAYS_INLINE void
```

Comment:
can u pls remind why need to add it to new protocols? could maybe just disable new protocol tests? or disable eager with new protocols (to keep only rndv)?